<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWidget</class>
 <widget class="QWidget" name="ConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="labelReplace">
       <property name="text">
        <string>Replace:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEditTarget"/>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="labelBy">
           <property name="text">
            <string>by:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditReplacement"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <widget class="QGroupBox" name="groupBoxPatternSyntax">
       <property name="title">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QRadioButton" name="radioButtonFixedString">
          <property name="text">
           <string>Fixed string</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="radioButtonRegExp">
          <property name="text">
           <string>Regular expressions</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QRadioButton" name="radioButtonWildcard">
          <property name="text">
           <string>Wildcard</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="checkBoxCaseSensitive">
       <property name="text">
        <string>Case sensitive</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Pattern syntax:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>175</width>
       <height>115</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditTarget</tabstop>
  <tabstop>lineEditReplacement</tabstop>
  <tabstop>radioButtonFixedString</tabstop>
  <tabstop>radioButtonRegExp</tabstop>
  <tabstop>radioButtonWildcard</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
